patch/0012-arm64-dts-marvell-Add-Supermicro-SSE-G3748-board.patch

```diff
@@ -0,0 +1,145 @@
+From a2c35e835144054d553469f055522b49dae03bd3 Mon Sep 17 00:00:00 2001
+From: SelvaSupermicro  <selvarajr@supermicro.com>
+Date: Thu, 15 Feb 2024 21:44:03 +0000
+Subject: [PATCH] arm64: dts: marvell: Add Supermicro SSE-G3748 board
+
+This change adds device tree files for Supermicro SSE-G3748 board.
+This dts file is derived from Marvell RD-AC5X board.
+- SPI flash partition is changed.
```

Comment:
How is it changed?

Reply:
We use different SPI flash partitions from what is used in Marvell RD-AC5X dts file.
Supermicro SSE-G3748 smc_sse-g3748.dts uses:
                partition@0 {
                        label = "spi_flash_part0";
                        reg = <0x0 0x200000>;
                };

                parition@1 {
                        label = "spi_flash_part1";
                        reg = <0x200000 0x10000>;
                };

                parition@2 {
                        label = "spi_flash_part2";
                        reg = <0x210000 0xC00000>;
                };

                parition@3 {
                        label = "spi_flash_part3";
                        reg = <0xE10000 0x1F0000>;
                };

Marvell RD-AC5X ac5-98dx35xx-rd.dts uses:
                partition@0 {
                        label = "spi_flash_part0";
                        reg = <0x0 0x800000>;
                };

                parition@1 {
                        label = "spi_flash_part1";
                        reg = <0x800000 0x700000>;
                };

                parition@2 {
                        label = "spi_flash_part2";
                        reg = <0xF00000 0x100000>;
                };